dockertest/output/texttable.py

```diff
@@ -135,6 +135,7 @@ class TextTable(MutableSet, Sequence):
     _rows = None
 
     def __init__(self, table, columnranges=None, header=None, tabledata=None):
+        # pylint: disable=W0231
```

Comment:
Hmm, the super-class is mutable, so it may be easier to call MutableSet.__init__(self), it's going to get items assigned below (anyway). That fixes the problem instead of ignoring it.